cfpb/consumerfinance.gov
# P4C: Convert expandables to tables

Convert expandables to tables on grad path tool review screen to improve usability.



## Changes

- Convert expandables to tables on review screen


## How to test this PR

1. Check out branch and run gulp
2.  Navigate to `Review your financing plan` screen in grad path tool
3. Verify data output in tables


## Screenshots


![localhost_8000_paying-for-college_your-financial-path-to-graduation__iped=110635 pid=4510-3 houp=onCampus typp=bachelors lenp=4 ratp=inState depp=dependent houx=1395 fdx=498 clhx=127 trnx=750 hltx=265 entx=185 retx=282 taxx=143 othx=328](https://user-images.githubusercontent.com/778171/112601707-0023eb00-8dd0-11eb-8215-012b650f11f5.png)


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
